autogluon/utils/try_import.py

```diff
@@ -82,3 +82,12 @@ def try_import_faiss():
         raise ImportError(
             "Unable to import dependency faiss"
             "A quick tip is to install via `pip install faiss-cpu`. ")
+
+def try_import_fastai_v1():
+    try:
+        from pkg_resources import parse_version  # pylint: disable=import-outside-toplevel
+        import fastai
+        fastai_version = parse_version(fastai.__version__)
+        assert parse_version('1.0.61') <= fastai_version < parse_version('2.0.0'), 'Currently, we only support 1.0.61<=gluonnlp<2.0.0'
+    except OSError as e:
+        raise ImportError("Import fastai failed.")
```

Comment:
Add instructions for pip install of fastai + ideal version.

Reply:
Added